docs/sources/configure-client/grafana-agent/java/_index.md

```diff
@@ -0,0 +1,265 @@
+---
+title: "Profiling Java via the Grafana Agent"
+menuTitle: "Profiling Java via the Grafana Agent"
+description: "Learn about using Grafana Agent for continuous profiling Java processes for performance optimization."
+weight: 20
+---
+
+# Profiling Java via the Grafana Agent
+
+Grafana Agent supports Java profiling in [Flow mode](/docs/agent/latest/flow/). The configuration file is written in
+the [River](/docs/agent/latest/flow/config-language/) language and is composed of components that are used to collect,
+transform, and send data.
+
+## Configuration
```

Comment:
I'm suggesting a verb for the heading since this section contains steps the user should do or verify. It's not a procedure per se (although consider making it a procedure in a future revision). 

Is this correct? Does this section just discussion components for the Agent or is there better wording? 
```suggestion
## Configure the components
```